packages/dashboards/package.json

```diff
@@ -85,10 +86,12 @@
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-ember": "^6.2.0",
     "eslint-plugin-node": "^8.0.1",
+    "faker": "^4.1.0",
     "loader.js": "^4.7.0",
     "navi-core": "^0.1.0",
     "navi-data": "^0.1.0",
     "navi-reports": "^0.1.0",
+    "pretender": "^3.0.2",
```

Comment:
Do we need this in `dashboards` and `reports`? It is a dep of `@miragejs/server`. I think we only import `pretender` explicitly in `navi-data`.

Reply:
fixed